conjure-go-client/httpclient/metrics.go

```diff
@@ -45,6 +47,9 @@ const (
 	CipherTagKey              = "cipher"
 	NextProtocolTagKey        = "next_protocol"
 	TLSVersionTagKey          = "tls_version"
+
+	MetricInFlightConns    = "client.conns.inflight"
+	MetricInFlightRequests = "client.request.inflight"
```

Comment:
Are these the same metric names used in conjure-java / dialogue?

Reply:
@carterkozak are there java equivalents for these? I'm thinking about changing them to `client.*.open(.count)`

Reply:
There are! https://github.com/palantir/dialogue/blob/develop/dialogue-clients/metrics.md#dialogueclientpool

Reply:
ok I tweaked what we have slightly to match verbiage but since we aren't able to introspect the pool itself, unfortunately we can't match the dialogue metrics exactly. I added `client.connection.create` with `reused:{true,false}` so we can know how many connections are being reused